plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -15,6 +15,8 @@ class Release < ActiveRecord::Base
 
     attr_accessor :builds
 
+    after_save :save_builds
```

Comment:
deploy_executor / whoever creates the release should assign deploy.builds just like job_execution does for a symmetric flow ?